src/Reflex/Process.hs

```diff
@@ -185,16 +179,17 @@ unsafeCreateProcessWithHandles
   -- the @e -> IO ()@ is a function that will trigger the output @Event t e@ when called. The resulting
   -- @IO ()@ will be run in a separate thread and must block until there is no more data in the 'Handle' to
   -- process.
+  -> Event t P.Signal
+  -- ^ Signals to send to the process.
```

Comment:
If we go this direction it might even make sense to move this up a level and then this function becomes exclusively "creation" and "output" handling.